git/merge-strategies.md

```diff
@@ -114,3 +114,51 @@ related. We discourage Strategy 2 if there is only a single commit to be merged,
 since the merge commit itself provides no additional information. **Note that
 merge commits can be lost in `git rebase` events and will not show up in `git
 blame` lists.**
+
+### Appendix: git commands
+
+So far, we've elided the commands one would use to create the various kinds of
+history, in order not to distract from the concepts. We include them here for
+completeness.
+
+To create a non-linearizable non-FF merge of `feature` into `master`, say:
+
+```
+git checkout master
+git merge --no-ff feature
+```
+
+To rebase feature onto the latest commit(s) of `master`, say:
+
+```
+git checkout feature
+git rebase master
+```
+
+To create a linearizable non-FF merge (assuming the above rebase step has
+already happened), do the same `--no-ff` merge as above:
+
+```
+git checkout master
+git merge --no-ff feature
+```
+
+To create an FF merge (assuming the above rebase step has already happened),
+say:
+
+```
+git checkout master
+git merge feature
+```
+
+If you're the thorough type, you can also pass the `--ff-only` option to `git
+merge`, which is the default.
+
+Typically, you will want to delete the `feature` branch (locally and on
+`origin`) after it has been merged into `master`. To do that, assuming you're
+already on the `master` branch, say:
+
+```
+git branch -d feature
+git push origin --delete feature
```

Comment:
I thought about it, but figured this one was more intention-revealing. But I suppose I can mention both as options, lest people think we only recommend using the one. Will change.